src/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ToggleStyled } from './Toggle.styles';
+import { ToggleProps } from './types';
+
+export const Toggle = ({ checked = false, disabled = false, size = 'medium', onChange, ...rest }: ToggleProps) => {
+  return (
+    <ToggleStyled size={size}>
+      <input
+        type="checkbox"
+        checked={checked}
+        disabled={disabled}
+        onChange={onChange}
```

Comment:
I am not sure, but in the storybook when I click on the toggle is not calling to onChage. Maybe this one should be onClick?

Reply:
I believe onChange is the correct event. There was an issue where that event was not triggered. That is why storybook was not able to detect the call.